content-guidelines/extension-subtitle.md

```diff
@@ -8,7 +8,7 @@ skip_index: true
 
 # Extension Subtitle
 
-The subtitle is one of your extension’s most important pieces of content. It’s the thing that shows up in search results, drawing a potential user’s attention. It makes your extension more discoverable. And, it’s used everywhere—in search results, throughout [addons.mozilla.org](https://addons.mozilla.org 'addons.mozilla.org'), and in the Firefox browser.
+The subtitle is one of your extension’s most important pieces of content. It’s the thing that shows up in search results, drawing a potential user’s attention. It makes your extension more discoverable. And, it’s used everywhere—in search results, throughout [addons.mozilla.org][https://addons.mozilla.org 'addons.mozilla.org'], and in the Firefox browser.
```

Comment:
This will disable the linking if you use `[]`

Reply:
@championshuttler can the () be enclosed inside the []? Will that preserve the links?

Reply:
@Romicodes I don't think so https://guides.github.com/features/mastering-markdown/

Reply:
@championshuttler adding  [addons.mozilla.org] : https://addons.mozilla.org at the very end of the markdown page?